.github/workflows/continuous-integration.yml

```
@@ -206,7 +206,8 @@ jobs:
         if: ${{ !cancelled() }}
         working-directory: "/var/www/glpi"
         run: |
-          bin/console database:install --ansi --no-interaction --db-name=glpi --db-host=db --db-user=root --db-password="" --strict-configuration
+          printenv
+          bin/console database:install --ansi --no-interaction --db-name=glpi --db-host=db --db-user=root --db-password="" --strict-configuration --env='testing'
```

This should not be necessary.

```suggestion
          bin/console database:install --ansi --no-interaction --db-name=glpi --db-host=db --db-user=root --db-password="" --strict-configuration
```